CMakeLists.txt

```diff
@@ -19,13 +19,20 @@
 #  Build the examples.
 #  Default=true
 #
+# -DKDChart_ROOT=[KDChart install path]
+#  Enable KDChart support.
+#  Default=off
+
 cmake_minimum_required(VERSION 2.8.12)
 if(POLICY CMP0020)
   cmake_policy(SET CMP0020 NEW)
 endif()
 if(POLICY CMP0025)
   cmake_policy(SET CMP0025 NEW)
 endif()
+if(POLICY CMP0074)
```

Comment:
add a comment that policy 0074 (since CMake 3.17) tells find_package() to look for <PackageName>_ROOT variables..  In particular KDChart_ROOT